ansible/group_vars/all

```diff
@@ -46,27 +55,14 @@ nginx_vhosts:
         auth_basic_user_file /usr/local/biblebox/etc/basicauth;
       }
 
-      location /shared/ {
```

Comment:
The way I am understanding this, it looks like the fallback to web root content is no longer happening with this change. The intention of that was to allow the device to be shipped with some on device content not on the usb stick that would be merged with whatever is already on the device.